src/assets/CurrencyRateController.ts

```diff
@@ -77,13 +85,17 @@ export class CurrencyRateController extends BaseController<CurrencyRateConfig, C
       disabled: true,
       interval: 180000,
       nativeCurrency: 'ETH',
+      includeUSDRate: false,
     };
     this.defaultState = {
       conversionDate: 0,
       conversionRate: 0,
       currentCurrency: this.defaultConfig.currentCurrency,
       nativeCurrency: this.defaultConfig.nativeCurrency,
     };
+    if (config?.includeUSDRate) {
+      this.defaultState.usdConversionRate = 0;
+    }
```

Comment:
We should default this to `0` regardless of what `config` is set to:

```suggestion
       usdConversionRate: 0,
    };
```

`config` can change at runtime, and if it does this default would never get set otherwise.